test/pytest/conftest.py

```diff
@@ -121,7 +121,7 @@ def harnesses():
     '''
     Automatically init at least two Pure phones
     '''
-    found_pures = serial.find_Pures()
+    found_pures = serial.find_Devices()
```

Comment:
so now it should be renamed :D
Also - wouldn't adding a Harmony on the devices lists break pure-specific tests?

Reply:
The method is renamed to indicate a new purpose - it now looks not only for Pure :)

If we run Pure-specific tests on Harmony they will fail, obviously. 
Making harness work with Harmony is not necessary for development, I just noticed the lack of it while testing CP-1103. This change can be postponed.

Reply:
i meant the variable's name :D

Reply:
Fixed :)